app/styles/ui/_configure-git-user.scss

```diff
@@ -1,6 +1,15 @@
 #configure-git-user {
   .form-component {
     margin: var(--spacing-double) 0;
+
+    // prevents jumpiness when switching between manual and github account options
+    .sign-in-form-inputs {
+      min-height: 150px;
+    }
+
+    .text-box-component {
+      flex: unset;
+    }
```

Comment:
Not necessary as part of this PR, I just noticed the inputs jump when switching radio buttons so I added this to stop that.